### DISCUSSION TITLE
BOOST hub, broadcast

### DISCUSSION THREAD

**Discussion by Eco-Beny** [pybricks/support#1810] [Q&A]
BOOST hub:
The programme loads normally from Pybick. Both outlets work if I connect a sensor or a motor (outlet C and D). Both built-in motors (A and B) also work.
However, BOOST does not send - broadcast a signal to the second hub (SPIKE Prime)  and does not receive a signal from the third hub (CITY). What could be the reason, besides a possible malfunction ?

**Comment by dlech** [comment]
The BOOST Move hub has a very limited Bluetooth chip. It can't be connected to a computer, be broadcasting and be observing all at the same time. For best results, try just doing one of these at a time.

**Comment by Eco-Beny** [comment]
Hi.

We try a lot of things...
BOOST can recive from SPIKE add also from CITY hub. But  BOOST just don't
send (brodcast) any data.
I was not sure anymore, so students trancript simple program from YouTube (
YOUTUBE.COM/watch?v=liv37b7PXUQ ).  Works from SPIKE to CITYand opossite.
Works if SPIKE or CITY transmit and BOOST recive. Don't work in opossite
direction, meaning if BOOST shoud transit.

V sob., 7. sep. 2024 17:40 je oseba David Lechner ***@***.***>
napisala:

**Reply by dlech** [reply]
Does it work if you disconnect the BOOST from the computer and use the button on the hub to start the program like shown in the video?

**Comment by BertLindeman** [comment]
You beat be David.


I also made the two programs from this you tube movie.

Using a City hub as receiver and a move hub as sender.
If the move hub is connected to the PC,
the Cityhub does **not receive** the data from the movehub.
If the movehub is **not** connected to the PC,
then the Cityhub does receive and the motor moves and the hub color switches.


##test scenario preparation 
1. connect the movehub to the PC
2. start the send program (to load it into the hub)
3. disconnect the movehub
4. connect the cityhub
5. start the receive program (to load it into the hub)
6. disconnect the cityhub

##test scenario - run
(Assume both hubs are ON and blinking blue.)
1. start the send program on the movehub by pressing the button
2. start the receive program on the cityhub by pressing the button

Observe the cityhub it should be blinking red/green and the motor on the A port should be moving.

Conclusion: the movehub needs to be **not connected** to a PC to successfully broadcast.

**Reply by dlech** [reply]
Yes, this is the expected behavior due to limitations of the Bluetooth chip in that hub. Probably something we should add to the documentation.

**Reply by BertLindeman** [reply]
Added support issue #1819
